operations/services.md

```diff
@@ -931,3 +931,75 @@ Creates a new order with the specified products and items. Only positive charges
 | --- | --- | --- | --- |
 | `OrderId` | string | required | Unique identifier of the created order. |
 
+## Get all companionships
+
+Returns all companionships based on customers, reservations or reservation groups. One of them must be specified in the request.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/companionships/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "CustomerIds": [
+        "35d4b117-4e60-44a3-9580-c582117eff98"
+    ],
+    "ReservationIds": [
+        "bfee2c44-1f84-4326-a862-5289598f6e2d"
+    ],
+    "ReservationGroupIds": [
+        "c704dff3-7811-4af7-a3a0-7b2b0635ac59"
+    ]
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `CustomerIds` | array of string | optional | Unique identifiers of [Customer](customers.md#customer)s. |
+| `ReservationIds` | array of string | optional | Unique identifiers of [Reservation](reservations.md#reservation)s. |
+| `ReservationGroupIds` | array of string | optional | Unique identifiers of [Reservation group](reservations.md#reservation-group)s. |
+| `Extent` | [Companionship extent](services.md#companionship-extent) | optional | Extent of data to be returned. E.g. it is possible to specify that together with the companionships, customers, reservations, and reservation groups should be also returned. If not specified, only `Companionships` are returned. |
+
+#### Companionship extent
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Customers` | bool | optional | Whether the response should contain customers. |
```

Comment:
Shouldn't we also have Companionships in the extent? Check other API calls and make it similar. If other API calls do not have the main entity in extent, then don't change anything.

Reply:
In reservations/getAll Reservations are in Extent.
In customers/getAll Customers are not in Extent.